core/storage/statistics/gae_models.py

```diff
@@ -1134,6 +1134,25 @@ def delete_playthroughs_multi(cls, playthrough_ids):
         cls.delete_multi(instances)
 
 
+class LearnerAnswerDetailModel(base_models.BaseModel):
+    """Model for storing the approach/response user enters when he
```

Comment:
Maybe change to "Model for storing the approach/response the users enter when they are asked for it" ?

Reply:
Okay

Reply:
Done